_health-care/_js/components/personal-information/VaMedicalFacility.jsx

```diff
@@ -21,26 +26,24 @@ class VaMedicalFacility extends React.Component {
     this.selectId = _.uniqueId('preferred-va-facility-');
   }
 
-  handleChange(domEvent) {
-    this.props.onValueChange(domEvent.target.value);
+  handleChange(update) {
+    this.props.onValueChange(update);
   }
 
-  // TODO: Add the actual list of medical facilities
-  // TODO: Filter list based on state selected?
   render() {
+    let clinicList = [];
+    if (this.props.facilityState) {
+      const selectedState = this.props.facilityState;
```

Comment:
If we're going to set a variable to `this.props.facilityState`, might as well move it outside of the `if` block, and then check for `if (selectedState)` instead.